test/integration-tests/Uniswap.test.ts

```diff
@@ -363,18 +366,10 @@ describe('Uniswap V2 and V3 Tests:', () => {
         planner.addCommand(CommandType.SWEEP, [ETH_ADDRESS, MSG_SENDER, 0])
 
         const { commands, inputs } = planner
-        const ethBalanceBeforeAlice = await ethers.provider.getBalance(alice.address)
-        const ethBalanceBeforeBob = await ethers.provider.getBalance(bob.address)
-        const receipt = await (await router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE)).wait()
-
-        const ethBalanceAfterAlice = await ethers.provider.getBalance(alice.address)
-        const ethBalanceAfterBob = await ethers.provider.getBalance(bob.address)
-        const gasSpent = receipt.gasUsed.mul(receipt.effectiveGasPrice)
-
-        const aliceFee = ethBalanceAfterAlice.sub(ethBalanceBeforeAlice)
-        const bobEarnings = ethBalanceAfterBob.sub(ethBalanceBeforeBob).add(gasSpent)
-
-        expect(aliceFee.add(bobEarnings).mul(ONE_PERCENT_BIPS).div(10000)).to.eq(aliceFee)
+        await expect(router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE)).to.changeEtherBalances(
+          [alice, bob],
```

Comment:
@zhongeric if you look here - this one is checking 2 addresses. One sent the tx and one didnt, so gas is included for one of them not the other. its great hahah

Reply:
wow thats so helpful